### ISSUE TITLE
Feature Request: Polygon Area Measurement using straight lines

### ISSUE THREAD

**Issue by Miguel-Figuerola** [cnr-isti-vclab/TagLab#49]
Hello,

Is it possible to draw a polygon using straight lines? I need to measure the total area of my orthomosaic to get the cover % of each coral tracing. My orthomosaic map uploads into taglab with a background, giving me a map area which is different to the actual orthomosaic where I am doing the coral planar area measurements. Any suggestions on how to get this area within TagLab?

I tried to get the area of the orthomosaic from Agisoft Metashape Pro by making a polygon shape, setting it as the outer boundaries of the orthomosaic, and using the measurement tool. However, I had to resize the .png file after exporting from Metashape because TagLab only goes up to 32767 pixels and my map was bigger. So I think when resizing the map, the area that Metashape gave me for the orthomosaic is no longer the correct one to get coral cover %.

So in summary TagLab only gives me the orthomosaic+the background area in pixels. Any suggestion on how to measure the area of a polygon using straight lines?

**Comment by gaiapavoni** [comment]
Dear Miguel, 
No, at the moment,  TagLab doesn't allow you to draw a polygon. But there are a couple of workarounds for your problem.
- If you want a squared area of interest, set it as a working area and then measure it using the ruler (we can think about adding its coverage to the export histogram function and the coverage percentage in the future). 
- if you need a more complicated polygon, you can try to draw it on Metashape or a GIS platform and export it as a shapefile. Import Shapefile>Import as a labelling layer (the reference layer will be activated in the next release).

Best, 

Gaia
